dashboard/modules/logical_view/test_logical_view_head.py

```diff
@@ -75,5 +74,66 @@ class InfeasibleActor:
                 raise Exception(f"Timed out while testing, {ex_stack}")
 
 
+def test_kill_actor(ray_start_with_dashboard):
+    @ray.remote
+    class Actor:
+        def __init__(self):
+            pass
+
+        def f(self):
+            ray.show_in_dashboard("test")
+            return os.getpid()
+
+    a = Actor.remote()
+    worker_pid = ray.get(a.f.remote())  # noqa
+
+    webui_url = ray_start_with_dashboard["webui_url"]
+    assert wait_until_server_available(webui_url)
+    webui_url = format_web_url(webui_url)
+
+    def actor_killed(PID):
+        """Check For the existence of a unix pid."""
+        try:
+            os.kill(PID, 0)
+        except OSError:
+            return True
+        else:
+            return False
+
+    def get_actor():
+        resp = requests.get(f"{webui_url}/logical/actor_groups")
+        resp.raise_for_status()
+        actor_groups_resp = resp.json()
+        assert actor_groups_resp["result"] is True, actor_groups_resp["msg"]
+        actor_groups = actor_groups_resp["data"]["actorGroups"]
+        actor = actor_groups["Actor"]["entries"][0]
+        return actor
+
+    def kill_actor(actor):
+        resp = requests.get(
+            webui_url + "/logical/kill_actor",
+            params={
+                "actorId": actor["actorId"],
+                "ipAddress": actor["ipAddress"],
+                "port": actor["port"]
+            })
+        resp.raise_for_status()
+        resp_json = resp.json()
+        assert resp_json["result"] is True, "msg" in resp_json
+
+    start = time.time()
+    last_exc = None
+    while time.time() - start <= 10:
+        try:
+            actor = get_actor()
+            kill_actor(actor)
+            last_exc = None
+            break
+        except (KeyError, AssertionError) as e:
+            last_exc = e
+            time.sleep(.1)
+    assert last_exc is None
```

Comment:
```suggestion
    assert last_exc is None, (f"last exception: {last_exc}")
```

Reply:
If I assert last_exc is None and last_exc is not None, it already prints an assertion error that says what last_exc was.

Reply:
oh but if last_exc is not None, this assertion will fail, and we'd like to print a better message right?